dotnet/src/SemanticKernel.Abstractions/IKernel.cs

```diff
@@ -52,12 +53,14 @@ public interface IKernel
     /// Run a pipeline composed of synchronous and asynchronous functions.
     /// </summary>
     /// <param name="variables">Input to process</param>
+    /// <param name="requestSettings">Request settings to use in the functions call</param>
     /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
     /// <param name="pipeline">List of functions</param>
     /// <returns>Result of the function composition</returns>
     Task<KernelResult> RunAsync(
         ContextVariables variables,
-        CancellationToken cancellationToken,
+        AIRequestSettings? requestSettings = null,
+        CancellationToken cancellationToken = default,
```

Comment:
Does this need to be tagged as a breaking change?

Or, can we add a method to `KernelExtensions` that matches the old signature?

Reply:
Added